### ISSUE TITLE
minishift start fails with default openshift 3.6.1 on macOS high sierra 10.13.2, 17.12.0-ce-mac47

### ISSUE THREAD

**Issue by starksm64** [minishift/minishift#1924]
### General information

  * Minishift version: minishift v1.11.0+4459917
  * OS: macOS high sierra 10.13.2
  * Hypervisor: xhyve: 1f1dbe

### Steps to reproduce

  1. Clear any existing ~/.minishift contents
  2. run minishift start

### Expected
OpenShift starts and shows the console login information

### Actual
```
[Istio 855]$ sudo rm -rf ~/.minishift/
[Istio 856]$ sudo rm -rf ~/.kube/
[Istio 857]$ minishift start
-- Starting profile 'minishift'
-- Checking if requested hypervisor 'xhyve' is supported on this platform ... OK
-- Checking if xhyve driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
   Checking for setuid bit ... OK
-- Checking the ISO URL ... OK
-- Starting local OpenShift cluster using 'xhyve' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB

   Downloading ISO 'https://github.com/minishift/minishift-b2d-iso/releases/download/v1.2.0/minishift-b2d.iso'
 40.00 MiB / 40.00 MiB [======================================================] 100.00% 0s
-- Starting Minishift VM ......................... OK
-- Checking for IP address ... OK
-- Checking if external host is reachable from the Minishift VM ... 
   Pinging 8.8.8.8 ... OK
-- Checking HTTP connectivity from the VM ... 
   Retrieving http://minishift.io/index.html ... OK
-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 0% used OK
-- Downloading OpenShift binary 'oc' version 'v3.6.1'
 33.79 MiB / 33.79 MiB [======================================================] 100.00% 0s-- Downloading OpenShift v3.6.1 checksums ... OK
-- OpenShift cluster will be configured with ...
   Version: v3.6.1
-- Checking `oc` support for startup flags ... 
   host-config-dir ... OK
   host-data-dir ... OK
   host-pv-dir ... OK
   host-volumes-dir ... OK
   routing-suffix ... OK
Starting OpenShift using openshift/origin:v3.6.1 ...
Pulling image openshift/origin:v3.6.1
Pulled 1/4 layers, 26% complete
Pulled 2/4 layers, 64% complete
Pulled 3/4 layers, 88% complete
Pulled 4/4 layers, 100% complete
Extracting
Image pull complete
-- Checking OpenShift client ... OK
-- Checking Docker client ... OK
-- Checking Docker version ... OK
-- Checking for existing OpenShift container ... OK
-- Checking for openshift/origin:v3.6.1 image ... 
   Pulling image openshift/origin:v3.6.1
   Pulled 1/4 layers, 26% complete
   Pulled 2/4 layers, 64% complete
   Pulled 3/4 layers, 88% complete
   Pulled 4/4 layers, 100% complete
   Extracting
   Image pull complete
-- Checking Docker daemon configuration ... OK
-- Checking for available ports ... OK
-- Checking type of volume mount ... 
   Using Docker shared volumes for OpenShift volumes
-- Creating host directories ... OK
-- Finding server IP ... 
   Using 192.168.64.6 as the server IP
-- Starting OpenShift container ... 
   Creating initial OpenShift configuration
   Starting OpenShift using container 'origin'
   Waiting for API server to start listening
FAIL
   Error: cannot access master readiness URL https://192.168.64.6:8443/healthz/ready
   Details:
     No log available from "origin" container

   Caused By:
     Error: Get https://192.168.64.6:8443/healthz/ready: read tcp 192.168.64.1:60223->192.168.64.6:8443: read: connection reset by peer
Error during 'cluster up' execution: Error starting the cluster.
[Istio 858]$ minishift status
Minishift:  Running
Profile:    minishift
OpenShift:  Stopped
DiskUsage:  7% of 17.9G
```

### Logs
https://gist.github.com/starksm64/4126bfca155e1692b7101ed0a87f14c5

**Comment by starksm64** [comment]
Note that if I set the config to use OpenShift 3.7.0 it starts fine:

[Istio 861]$ sudo rm -rf ~/.minishift/
[Istio 862]$ minishift config set openshift-version v3.7.0
[Istio 863]$ minishift start
-- Starting profile 'minishift'
-- Checking if requested hypervisor 'xhyve' is supported on this platform ... OK
-- Checking if xhyve driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
   Checking for setuid bit ... OK
-- Checking the ISO URL ... OK
-- Starting local OpenShift cluster using 'xhyve' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB

   Downloading ISO 'https://github.com/minishift/minishift-b2d-iso/releases/download/v1.2.0/minishift-b2d.iso'
 40.00 MiB / 40.00 MiB [======================================================] 100.00% 0s
-- Starting Minishift VM ......................... OK
-- Checking for IP address ... OK
-- Checking if external host is reachable from the Minishift VM ... 
   Pinging 8.8.8.8 ... OK
-- Checking HTTP connectivity from the VM ... 
   Retrieving http://minishift.io/index.html ... OK
-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 0% used OK
-- Downloading OpenShift binary 'oc' version 'v3.7.0'
 38.18 MiB / 38.18 MiB [======================================================] 100.00% 0s-- Downloading OpenShift v3.7.0 checksums ... OK
-- OpenShift cluster will be configured with ...
   Version: v3.7.0
-- Checking `oc` support for startup flags ... 
   routing-suffix ... OK
   host-config-dir ... OK
   host-data-dir ... OK
   host-pv-dir ... OK
   host-volumes-dir ... OK
   version ... OK
Starting OpenShift using openshift/origin:v3.7.0 ...
Pulling image openshift/origin:v3.7.0
Pulled 1/4 layers, 26% complete
Pulled 2/4 layers, 73% complete
Pulled 3/4 layers, 86% complete
Pulled 4/4 layers, 100% complete
Extracting
Image pull complete
OpenShift server started.

The server is accessible via web console at:
    https://192.168.64.7:8443

You are logged in as:
    User:     developer
    Password: <any value>

To login as administrator:
    oc login -u system:admin

[Istio 864]$

**Comment by praveenkumar** [comment]
I tried on high sierra and didn't encounter this issue.

```
$ system_profiler SPSoftwareDataType
Software:

    System Software Overview:

      System Version: macOS 10.13.2 (17C88)
      Kernel Version: Darwin 17.3.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 6 minutes

$ ./minishift start 
[...]
-- Downloading OpenShift binary 'oc' version 'v3.6.1'
 33.79 MiB / 33.79 MiB [=================================================================================================================================================] 100.00% 0s-- Downloading OpenShift v3.6.1 checksums ... OK
-- OpenShift cluster will be configured with ...
   Version: v3.6.1
-- Checking `oc` support for startup flags ... 
   host-pv-dir ... OK
   host-volumes-dir ... OK
   routing-suffix ... OK
   host-config-dir ... OK
   host-data-dir ... OK
Starting OpenShift using openshift/origin:v3.6.1 ...
Pulling image openshift/origin:v3.6.1
Pulled 1/4 layers, 26% complete
Pulled 1/4 layers, 68% complete
Pulled 2/4 layers, 86% complete
Pulled 3/4 layers, 92% complete
Pulled 4/4 layers, 100% complete
Extracting
Image pull complete
OpenShift server started.

The server is accessible via web console at:
    https://192.168.64.83:8443

You are logged in as:
    User:     developer
    Password: <any value>

To login as administrator:
    oc login -u system:admin
```

**Comment by chargio** [comment]
What you are describing is exactly the issue.
-- Downloading OpenShift binary 'oc' version 'v3.6.1'

I get thesmae behaviour, and when I updated to minishift 1.11.0 it deleted my config so it went back to 3.6.1.

**Comment by gbraad** [comment]
Unable to reproduce here. Can you provide more information about the setup,
like versions, steps taken, etc.

-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by chargio** [comment]
```
brew install minishift
minishift start.
```
You can see the messages everyone here is receiving: v3.6.1 is installed, and not v.3.7.1

**Comment by gbraad** [comment]
Sorry, I mean version of the OS, patch level, any other things you might
run, like Docker for Mac, etc. I want to be sure it is reproducible for the
same versions as in the title.

-- 

   Gerard Braad | http://gbraad.nl
   [ Doing Open Source Matters ]

**Comment by starksm64** [comment]
Here is all of my version information:

[Istio 876]$ system_profiler SPSoftwareDataType
Software:

    System Software Overview:

      System Version: macOS 10.13.2 (17C205)
      Kernel Version: Darwin 17.3.0
      Boot Volume: MacOSXRoot
      Boot Mode: Normal
      Computer Name: ScottOfficeMacPro
      User Name: Scott M Stark (starksm)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 4 days 2:36
[Istio 877]$ minishift version
minishift v1.11.0+4459917
[Istio 878]$ xhyve -v
xhyve: 1f1dbe

xhyve is a port of FreeBSD's bhyve hypervisor to OS X that
works entirely in userspace and has no other dependencies.

Homepage: https://github.com/mist64/xhyve
License: BSD

[istio_tutorial 1142]$ docker version
Client:
 Version:	17.12.0-ce
 API version:	1.35
 Go version:	go1.9.2
 Git commit:	c97c6d6
 Built:	Wed Dec 27 20:03:51 2017
 OS/Arch:	darwin/amd64

Server:
 Engine:
  Version:	17.12.0-ce
  API version:	1.35 (minimum version 1.12)
  Go version:	go1.9.2
  Git commit:	c97c6d6
  Built:	Wed Dec 27 20:12:29 2017
  OS/Arch:	linux/amd64
  Experimental:	true

**Comment by chargio** [comment]
```
Software:

    System Software Overview:

      System Version: macOS 10.13.2 (17C205)
      Kernel Version: Darwin 17.3.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: sergio-OSX
      User Name: Sergio Ocón (sergio)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 1:58
```

```
$ minishift version
minishift v1.11.0+4459917
```

```
$ xhyve -v
xhyve: 1f1db
```

**Comment by gbraad** [comment]
Thanks for the info. We will need to investigate this more

**Comment by gbraad** [comment]
Can you try to use a released binary from our GitHub releases? https://github.com/minishift/minishift/releases/tag/v1.12.0

We do not support installs performed with Brew.

**Comment by starksm64** [comment]
I have downloaded the v1.12.0 and cleared all of the existing ~/.minishift and ~/.kube directories.

Running `minishift start` looks to be failing to start openshift still:

`[bin 1225]$ minishift version
minishift v1.12.0+daa0943
[bin 1226]$ minishift start
-- Starting profile 'minishift'
-- Checking if requested hypervisor 'xhyve' is supported on this platform ... OK
-- Checking if xhyve driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
   Checking for setuid bit ... OK
-- Checking the ISO URL ... OK
-- Starting local OpenShift cluster using 'xhyve' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB

   Downloading ISO 'https://github.com/minishift/minishift-b2d-iso/releases/download/v1.2.0/minishift-b2d.iso'
 40.00 MiB / 40.00 MiB [======================================================] 100.00% 0s
-- Starting Minishift VM ......................... OK
-- Checking for IP address ... OK
-- Checking if external host is reachable from the Minishift VM ... 
   Pinging 8.8.8.8 ... OK
-- Checking HTTP connectivity from the VM ... 
   Retrieving http://minishift.io/index.html ... OK
-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 0% used OK
-- Downloading OpenShift binary 'oc' version 'v3.6.1'
 33.79 MiB / 33.79 MiB [======================================================] 100.00% 0s-- Downloading OpenShift v3.6.1 checksums ... OK
-- OpenShift cluster will be configured with ...
   Version: v3.6.1
-- Checking `oc` support for startup flags ... 
   host-data-dir ... OK
   host-pv-dir ... OK
   host-volumes-dir ... OK
   routing-suffix ... OK
   host-config-dir ... OK
Starting OpenShift using openshift/origin:v3.6.1 ...
Pulling image openshift/origin:v3.6.1
Pulled 1/4 layers, 26% complete
Pulled 2/4 layers, 53% complete
Pulled 3/4 layers, 82% complete
Pulled 4/4 layers, 100% complete
Extracting
Image pull complete
-- Checking OpenShift client ... OK
-- Checking Docker client ... OK
-- Checking Docker version ... OK
-- Checking for existing OpenShift container ... OK
-- Checking for openshift/origin:v3.6.1 image ... 
   Pulling image openshift/origin:v3.6.1
   Pulled 1/4 layers, 26% complete
   Pulled 2/4 layers, 53% complete
   Pulled 3/4 layers, 82% complete
   Pulled 4/4 layers, 100% complete
   Extracting
   Image pull complete
-- Checking Docker daemon configuration ... OK
-- Checking for available ports ... OK
-- Checking type of volume mount ... 
   Using Docker shared volumes for OpenShift volumes
-- Creating host directories ... OK
-- Finding server IP ... 
   Using 192.168.64.13 as the server IP
-- Starting OpenShift container ... 
   Creating initial OpenShift configuration
   Starting OpenShift using container 'origin'
   Waiting for API server to start listening
FAIL
   Error: cannot access master readiness URL https://192.168.64.13:8443/healthz/ready
   Details:
     No log available from "origin" container

   Caused By:
     Error: Get https://192.168.64.13:8443/healthz/ready: dial tcp 192.168.64.13:8443: getsockopt: connection refused
Error during 'cluster up' execution: Error starting the cluster.
[bin 1229]$ minishift status
Minishift:  Running
Profile:    minishift
OpenShift:  Stopped
DiskUsage:  7% of 17.9G
[bin 1230]$ minishift logs
W0125 10:35:03.433771    2999 start_master.go:297] Warning: assetConfig.loggingPublicURL: Invalid value: "": required to view aggregated container logs in the console, master start will continue.
W0125 10:35:03.433992    2999 start_master.go:297] Warning: assetConfig.metricsPublicURL: Invalid value: "": required to view cluster metrics in the console, master start will continue.
W0125 10:35:03.434005    2999 start_master.go:297] Warning: auditConfig.auditFilePath: Required value: audit can not be logged to a separate file, master start will continue.
I0125 10:35:03.447865    2999 plugins.go:101] No cloud provider specified.
E0125 10:35:03.459650    2999 controllermanager.go:337] Server isn't healthy yet.  Waiting a little while.
2018-01-25 10:35:03.466347 I | etcdserver/api/v3rpc: grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:4001: getsockopt: connection refused"; Reconnecting to {127.0.0.1:4001 <nil>}
2018-01-25 10:35:03.474183 I | etcdserver/api/v3rpc: grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:4001: getsockopt: connection refused"; Reconnecting to {127.0.0.1:4001 <nil>}
2018-01-25 10:35:03.475427 I | etcdserver/api/v3rpc: grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:4001: getsockopt: connection refused"; Reconnecting to {127.0.0.1:4001 <nil>}
2018-01-25 10:35:03.476593 I | etcdserver/api/v3rpc: grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:4001: getsockopt: connection refused"; Reconnecting to {127.0.0.1:4001 <nil>}
2018-01-25 10:35:03.477742 I | etcdserver/api/v3rpc: grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:4001: getsockopt: connection refused"; Reconnecting to {127.0.0.1:4001 <nil>}
I0125 10:35:04.119888    2999 master_config.go:406] Will report 192.168.64.13 as public IP address.
2018-01-25 10:35:04.130657 I | etcdserver/api/v3rpc: grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:4001: getsockopt: connection refused"; Reconnecting to {127.0.0.1:4001 <nil>}
I0125 10:35:04.131352    2999 start_master.go:529] Starting master on 0.0.0.0:8443 (v3.6.1+008f2d5)
I0125 10:35:04.131666    2999 start_master.go:530] Public master address is https://192.168.64.13:8443
I0125 10:35:04.131703    2999 start_master.go:534] Using images from "openshift/origin-<component>:v3.6.1"
2018-01-25 10:35:04.131904 I | etcdserver/api/v3rpc: grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:4001: getsockopt: connection refused"; Reconnecting to {127.0.0.1:4001 <nil>}
2018-01-25 10:35:04.132207 I | embed: peerTLS: cert = /var/lib/origin/openshift.local.config/master/etcd.server.crt, key = /var/lib/origin/openshift.local.config/master/etcd.server.key, ca = /var/lib/origin/openshift.local.config/master/ca.crt, trusted-ca = , client-cert-auth = true
2018-01-25 10:35:04.133320 I | embed: listening for peers on https://0.0.0.0:7001
2018-01-25 10:35:04.133422 I | embed: listening for client requests on 0.0.0.0:4001
2018-01-25 10:35:04.325979 I | etcdserver: name = openshift.local
2018-01-25 10:35:04.326092 I | etcdserver: data dir = /var/lib/origin/openshift.local.etcd
2018-01-25 10:35:04.326116 I | etcdserver: member dir = /var/lib/origin/openshift.local.etcd/member
2018-01-25 10:35:04.326132 I | etcdserver: heartbeat = 100ms
2018-01-25 10:35:04.326146 I | etcdserver: election = 1000ms
2018-01-25 10:35:04.326161 I | etcdserver: snapshot count = 100000
2018-01-25 10:35:04.326612 I | etcdserver: advertise client URLs = https://127.0.0.1:4001
2018-01-25 10:35:04.326743 I | etcdserver: initial advertise peer URLs = https://127.0.0.1:7001
2018-01-25 10:35:04.326856 I | etcdserver: initial cluster = openshift.local=https://127.0.0.1:7001
E0125 10:35:04.464805    2999 controllermanager.go:337] Server isn't healthy yet.  Waiting a little while.
2018-01-25 10:35:04.670506 I | etcdserver: starting member 51cc720fdd39e048 in cluster dcf5ba954f7ebe11
2018-01-25 10:35:04.671252 I | raft: 51cc720fdd39e048 became follower at term 0
2018-01-25 10:35:04.671386 I | raft: newRaft 51cc720fdd39e048 [peers: [], term: 0, commit: 0, applied: 0, lastindex: 0, lastterm: 0]
2018-01-25 10:35:04.671442 I | raft: 51cc720fdd39e048 became follower at term 1
2018-01-25 10:35:05.128952 W | auth: simple token is not cryptographically signed
2018-01-25 10:35:05.316024 I | etcdserver: starting server... [version: 3.2.1, cluster version: to_be_decided]
2018-01-25 10:35:05.316325 I | embed: ClientTLS: cert = /var/lib/origin/openshift.local.config/master/etcd.server.crt, key = /var/lib/origin/openshift.local.config/master/etcd.server.key, ca = /var/lib/origin/openshift.local.config/master/ca.crt, trusted-ca = , client-cert-auth = true
2018-01-25 10:35:05.317875 I | etcdserver/membership: added member 51cc720fdd39e048 [https://127.0.0.1:7001] to cluster dcf5ba954f7ebe11
E0125 10:35:05.461077    2999 controllermanager.go:337] Server isn't healthy yet.  Waiting a little while.
2018-01-25 10:35:05.872984 I | raft: 51cc720fdd39e048 is starting a new election at term 1
2018-01-25 10:35:05.873116 I | raft: 51cc720fdd39e048 became candidate at term 2
2018-01-25 10:35:05.873156 I | raft: 51cc720fdd39e048 received MsgVoteResp from 51cc720fdd39e048 at term 2
2018-01-25 10:35:05.873188 I | raft: 51cc720fdd39e048 became leader at term 2
2018-01-25 10:35:05.873204 I | raft: raft.node: 51cc720fdd39e048 elected leader 51cc720fdd39e048 at term 2
2018-01-25 10:35:05.873989 I | etcdserver: setting up the initial cluster version to 3.2
2018-01-25 10:35:05.910230 N | etcdserver/membership: set the initial cluster version to 3.2
2018-01-25 10:35:05.910837 I | etcdserver/api: enabled capabilities for version 3.2
2018-01-25 10:35:05.911007 I | embed: ready to serve client requests
2018-01-25 10:35:05.911116 I | etcdserver: published {Name:openshift.local ClientURLs:[https://127.0.0.1:4001]} to cluster dcf5ba954f7ebe11
I0125 10:35:05.911258    2999 run.go:85] Started etcd at 127.0.0.1:4001
2018-01-25 10:35:05.911889 I | embed: serving client requests on [::]:4001
2018-01-25 10:35:06.114209 I | etcdserver/api/v3rpc: Failed to dial [::]:4001: connection error: desc = "transport: remote error: tls: bad certificate"; please retry.
2018-01-25 10:35:06.125474 I | etcdserver/api/v3rpc: Failed to dial [::]:4001: connection error: desc = "transport: remote error: tls: bad certificate"; please retry.
2018-01-25 10:35:06.131118 I | etcdserver/api/v3rpc: Failed to dial [::]:4001: connection error: desc = "transport: remote error: tls: bad certificate"; please retry.
2018-01-25 10:35:06.131441 I | etcdserver/api/v3rpc: Failed to dial [::]:4001: connection error: desc = "transport: remote error: tls: bad certificate"; please retry.
2018-01-25 10:35:06.131740 I | etcdserver/api/v3rpc: Failed to dial [::]:4001: connection error: desc = "transport: remote error: tls: bad certificate"; please retry.
2018-01-25 10:35:06.134100 I | etcdserver/api/v3rpc: Failed to dial [::]:4001: connection error: desc = "transport: remote error: tls: bad certificate"; please retry.
2018-01-25 10:35:06.135237 I | etcdserver/api/v3rpc: Failed to dial [::]:4001: connection error: desc = "transport: remote error: tls: bad certificate"; please retry.
2018-01-25 10:35:06.135395 I | etcdserver/api/v3rpc: Failed to dial [::]:4001: connection error: desc = "transport: remote error: tls: bad certificate"; please retry.
I0125 10:35:06.135906    2999 run_components.go:91] Using default project node label selector: 
I0125 10:35:06.139588    2999 clusterquotamapping.go:160] Starting ClusterQuotaMappingController controller
E0125 10:35:06.139928    2999 reflector.go:201] github.com/openshift/origin/pkg/authorization/generated/informers/internalversion/factory.go:45: Failed to list *authorization.PolicyBinding: Get https://127.0.0.1:8443/apis/authorization.openshift.io/v1/policybindings?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
E0125 10:35:06.140361    2999 reflector.go:201] github.com/openshift/origin/pkg/authorization/generated/informers/internalversion/factory.go:45: Failed to list *authorization.ClusterPolicy: Get https://127.0.0.1:8443/apis/authorization.openshift.io/v1/clusterpolicies?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
E0125 10:35:06.140593    2999 reflector.go:201] github.com/openshift/origin/pkg/authorization/generated/informers/internalversion/factory.go:45: Failed to list *authorization.Policy: Get https://127.0.0.1:8443/apis/authorization.openshift.io/v1/policies?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
I0125 10:35:06.140767    2999 master.go:182] Starting OAuth2 API at /oauth
I0125 10:35:06.140917    2999 master.go:190] Starting Web Console /console/
E0125 10:35:06.141376    2999 reflector.go:201] github.com/openshift/origin/pkg/quota/generated/informers/internalversion/factory.go:45: Failed to list *quota.ClusterResourceQuota: Get https://127.0.0.1:8443/apis/quota.openshift.io/v1/clusterresourcequotas?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
E0125 10:35:06.141551    2999 reflector.go:201] github.com/openshift/origin/pkg/authorization/generated/informers/internalversion/factory.go:45: Failed to list *authorization.ClusterPolicyBinding: Get https://127.0.0.1:8443/apis/authorization.openshift.io/v1/clusterpolicybindings?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
2018-01-25 10:35:06.381127 W | etcdserver: apply entries took too long [126.418541ms for 1 entries]
2018-01-25 10:35:06.381422 W | etcdserver: avoid queries with large range/delete range!
E0125 10:35:06.467830    2999 controllermanager.go:337] Server isn't healthy yet.  Waiting a little while.
W0125 10:35:06.683850    2999 genericapiserver.go:295] Skipping API autoscaling/v2alpha1 because it has no resources.
W0125 10:35:06.920540    2999 genericapiserver.go:295] Skipping API rbac.authorization.k8s.io/v1alpha1 because it has no resources.
I0125 10:35:07.024809    2999 openshift_apiserver.go:237] Starting Origin API at /apis/apps.openshift.io/v1
I0125 10:35:07.027599    2999 openshift_apiserver.go:237] Starting Origin API at /apis/template.openshift.io/v1
I0125 10:35:07.033655    2999 openshift_apiserver.go:237] Starting Origin API at /apis/build.openshift.io/v1
I0125 10:35:07.037636    2999 openshift_apiserver.go:237] Starting Origin API at /apis/network.openshift.io/v1
I0125 10:35:07.039282    2999 openshift_apiserver.go:237] Starting Origin API at /apis/route.openshift.io/v1
I0125 10:35:07.095917    2999 openshift_apiserver.go:237] Starting Origin API at /apis/user.openshift.io/v1
I0125 10:35:07.212643    2999 openshift_apiserver.go:237] Starting Origin API at /apis/image.openshift.io/v1
I0125 10:35:07.216023    2999 openshift_apiserver.go:237] Starting Origin API at /apis/security.openshift.io/v1
I0125 10:35:07.217808    2999 openshift_apiserver.go:237] Starting Origin API at /apis/project.openshift.io/v1
I0125 10:35:07.220346    2999 openshift_apiserver.go:237] Starting Origin API at /apis/quota.openshift.io/v1
E0125 10:35:07.243737    2999 reflector.go:201] github.com/openshift/origin/pkg/authorization/generated/informers/internalversion/factory.go:45: Failed to list *authorization.ClusterPolicy: Get https://127.0.0.1:8443/apis/authorization.openshift.io/v1/clusterpolicies?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
E0125 10:35:07.244298    2999 reflector.go:201] github.com/openshift/origin/pkg/quota/generated/informers/internalversion/factory.go:45: Failed to list *quota.ClusterResourceQuota: Get https://127.0.0.1:8443/apis/quota.openshift.io/v1/clusterresourcequotas?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
E0125 10:35:07.244565    2999 reflector.go:201] github.com/openshift/origin/pkg/authorization/generated/informers/internalversion/factory.go:45: Failed to list *authorization.Policy: Get https://127.0.0.1:8443/apis/authorization.openshift.io/v1/policies?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
E0125 10:35:07.246591    2999 reflector.go:201] github.com/openshift/origin/pkg/authorization/generated/informers/internalversion/factory.go:45: Failed to list *authorization.PolicyBinding: Get https://127.0.0.1:8443/apis/authorization.openshift.io/v1/policybindings?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
E0125 10:35:07.246845    2999 reflector.go:201] github.com/openshift/origin/pkg/authorization/generated/informers/internalversion/factory.go:45: Failed to list *authorization.ClusterPolicyBinding: Get https://127.0.0.1:8443/apis/authorization.openshift.io/v1/clusterpolicybindings?resourceVersion=0: dial tcp 127.0.0.1:8443: getsockopt: connection refused
I0125 10:35:07.263050    2999 openshift_apiserver.go:237] Starting Origin API at /apis/authorization.openshift.io/v1
I0125 10:35:07.268859    2999 openshift_apiserver.go:237] Starting Origin API at /apis/oauth.openshift.io/v1
I0125 10:35:07.302348    2999 openshift_apiserver.go:243] Started Origin API at /oapi/v1
[restful] 2018/01/25 10:35:07 log.go:30: [restful/swagger] listing is available at https://192.168.64.13:8443/swaggerapi
[restful] 2018/01/25 10:35:07 log.go:30: [restful/swagger] https://192.168.64.13:8443/swaggerui/ is mapped to folder /swagger-ui/
E0125 10:35:07.469432    2999 controllermanager.go:337] Server isn't healthy yet.  Waiting a little while.
I0125 10:35:07.634228    2999 serve.go:86] Serving securely on 0.0.0.0:8443
W0125 10:35:07.753176    2999 run_components.go:60] Binding DNS on port 8053 instead of 53, which may not be resolvable from all clients
I0125 10:35:07.754308    2999 logs.go:41] skydns: ready for queries on cluster.local. for tcp4://0.0.0.0:8053 [rcache 0]
I0125 10:35:07.754534    2999 logs.go:41] skydns: ready for queries on cluster.local. for udp4://0.0.0.0:8053 [rcache 0]
I0125 10:35:07.856876    2999 run_components.go:86] DNS listening at 0.0.0.0:8053
I0125 10:35:08.047017    2999 ensure.go:231] No cluster policy found.  Creating bootstrap policy based on: /var/lib/origin/openshift.local.config/master/policy.json
E0125 10:35:08.463331    2999 controllermanager.go:337] Server isn't healthy yet.  Waiting a little while.
2018-01-25 10:35:08.974933 W | etcdserver: apply entries took too long [109.329146ms for 1 entries]
2018-01-25 10:35:08.975550 W | etcdserver: avoid queries with large range/delete range!
I0125 10:35:08.980336    2999 trace.go:76] Trace[1569405607]: "GuaranteedUpdate etcd3: *api.Endpoints" (started: 2018-01-25 10:35:08.428016165 +0000 UTC) (total time: 552.155223ms):
Trace[1569405607]: [379.635031ms] [379.635031ms] initial value restored
Trace[1569405607]: [549.023265ms] [169.388234ms] Transaction prepared
W0125 10:35:09.314810    2999 lease_endpoint_reconciler.go:176] Resetting endpoints for master service "kubernetes" to [192.168.64.13]
E0125 10:35:09.463277    2999 controllermanager.go:337] Server isn't healthy yet.  Waiting a little while.
2018-01-25 10:35:09.684675 W | etcdserver: apply entries took too long [244.051508ms for 2 entries]
2018-01-25 10:35:09.684849 W | etcdserver: avoid queries with large range/delete range!
I0125 10:35:09.687780    2999 trace.go:76] Trace[1591726117]: "Create /api/v1/namespaces/kube-system/configmaps" (started: 2018-01-25 10:35:08.978679451 +0000 UTC) (total time: 708.986152ms):
Trace[1591726117]: [339.261254ms] [339.095863ms] About to store object in database
Trace[1591726117]: [708.764226ms] [369.502972ms] Object stored in database
I0125 10:35:10.503937    2999 start_master.go:715] Started serviceaccount-token controller
2018-01-25 10:35:10.654807 W | etcdserver: apply entries took too long [176.66045ms for 1 entries]
2018-01-25 10:35:10.655015 W | etcdserver: avoid queries with large range/delete range!
2018-01-25 10:35:11.085446 W | etcdserver: apply entries took too long [241.353587ms for 1 entries]
2018-01-25 10:35:11.085657 W | etcdserver: avoid queries with large range/delete range!
I0125 10:35:11.691332    2999 trace.go:76] Trace[1528326197]: "Get /api/v1/namespaces/kube-system/serviceaccounts/generic-garbage-collector" (started: 2018-01-25 10:35:11.090926986 +0000 UTC) (total time: 600.353152ms):
Trace[1528326197]: [600.353152ms] [600.353152ms] END
I0125 10:35:11.692451    2999 trace.go:76] Trace[1341601819]: "List /api/v1/namespaces/kube-system/secrets" (started: 2018-01-25 10:35:11.09433759 +0000 UTC) (total time: 598.079014ms):
Trace[1341601819]: [597.806259ms] [597.71395ms] Listing from storage done
2018-01-25 10:35:12.054430 W | etcdserver: apply entries took too long [235.520408ms for 1 entries]
2018-01-25 10:35:12.054487 W | etcdserver: avoid queries with large range/delete range!
2018-01-25 10:35:24.392669 W | etcdserver: apply entries took too long [114.611726ms for 1 entries]
2018-01-25 10:35:24.392865 W | etcdserver: avoid queries with large range/delete range!
2018-01-25 10:35:25.997676 W | etcdserver: apply entries took too long [109.92603ms for 1 entries]
2018-01-25 10:35:25.997983 W | etcdserver: avoid queries with large range/delete range!
2018-01-25 10:35:37.080683 W | etcdserver: apply entries took too long [176.837025ms for 1 entries]
2018-01-25 10:35:37.080989 W | etcdserver: avoid queries with large range/delete range!
F0125 10:35:42.534125    2999 start_master.go:506] timed out waiting for the condition

[bin 1231]$ 

`
